### ISSUE TITLE
Translations will be lost on next publication from master.

### ISSUE THREAD

**Issue by zepumph** [phetsims/molarity#235]
Over in #234, we removed sim-specific content, including string keys in molarity, in exchange for new ones over in scenery-phet for a common code component. This issue is supposed to be similar to https://github.com/phetsims/gravity-force-lab/issues/166 in that molarity-specific translations will be no longer used as those keys are deleted.

I think there is a good chance though that by the time this is published again, the scenery-phet strings will all be translated from use in another sim, perhaps there will be some automatic work done on that front. Noting this for next release.

**Comment by zepumph** [comment]
Also tagging @jbphet because he is our string guy!

**Comment by jbphet** [comment]
Thanks for the tag.  

Before the next release, I'd recommend looking through the changes and seeing how many translated strings were lost.  If it's a bunch, we can manually port the strings into their scenery-phet equivalents.  It seems like a shame to lose a bunch of work done by our trusted translators.

**Comment by zepumph** [comment]
It is 4 strings, as seen in https://github.com/phetsims/molarity/commit/bed5c0f41c36e01bbb129eaf073442040ae84188.

(8 total), but 4 are in the PDOM.

These 4 strings are not just blanketly moved to common code, but converted into patterns that are filled in based on the the object the combo box is changing ("Solutes" in molarity's case, "Challenges" in RaP).

I don't think it will be possible to convert, but maybe it won't be too hard to ask them to patch them in to the new version?

**Comment by jbphet** [comment]
>  [M]aybe it won't be too hard to ask them to patch them in to the new version?

Yes, we have done that before.  We can give @oliver-phet a list of the affected locales and and explanation of what has changed and he'll communicate with the trusted translators.
